### PR TITLE
Help center content update

### PR DESCRIPTION
**Pull request by jwhrec**
## Description of the change

> Updated help center content for all pages except bulk upload

### Development

**This box MUST be checked by the submitter prior to merging**:
- [x] **Double- and triple-checked that there is no Personally Identifiable Information (PII) being mistakenly added in this pull request**

These boxes should be checked by the submitter prior to merging:
- [ ] Tests have been written to cover the code changed/added as part of this pull request

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: publisher/src/components/HelpCenter/Guides/AccountSetupGuide.tsx [hunk 1]
```diff
@@ -23,19 +23,26 @@ export const AccountSetupGuide = () => (
   <>
     <Styled.SectionWrapper>
       <Styled.SectionParagraph>
-        Within agency settings you can update information about your agency
-        including your URL and the description of your agency.
+        Within agency settings you can update:
+        <ul>
+          <li>agency users</li>
+          <li>agency url</li>
+          <li>agency url</li>
```
Comment (morden35) [inline publisher/src/components/HelpCenter/Guides/AccountSetupGuide.tsx:30]: nit: is this suppose to be here twice?

Comment (jwhrec) [reply publisher/src/components/HelpCenter/Guides/AccountSetupGuide.tsx:30]: nope - good catch!

Comment (jwhrec) [reply publisher/src/components/HelpCenter/Guides/AccountSetupGuide.tsx:30]: fixed!

File: publisher/src/components/HelpCenter/Guides/ManualEntryGuide.tsx [hunk 2]
```diff
@@ -31,23 +31,41 @@ export const ManualEntryGuide = () => (
...
-        To create a new record, click the create record button in the top right
-        corner. Then, specify the reporting period (either month or year) and
-        the starting period for the record. Next, click create record. The
-        record will be populated with metrics that have the matching
-        availability.
+        To create a new record, click <strong>New Record</strong> in the top
+        right corner. Then, specify the record period (either month or year) and
+        the starting period for the record. Next, click{" "}
+        <strong>Create Record</strong>. The record will be ready to accept data
+        for metrics that have the matching availability. availability.
```
Comment (morden35) [inline publisher/src/components/HelpCenter/Guides/ManualEntryGuide.tsx:68]: nit: double 'availability'

File: publisher/src/components/HelpCenter/Guides/SetUpMetricsGuide.tsx [hunk 3]
```diff
@@ -26,51 +26,46 @@ export const SetUpMetricsGuide = () => (
...
+      <Styled.SectionTitle>Metric Breakdowns</Styled.SectionTitle>
       <Styled.SectionParagraph>
-        Some metrics contain breakdown categories that help further define what
-        makes up the overall metric. For example, a <strong>Staff</strong>{" "}
-        metric could be broken down into{" "}
+        Some metrics provide the option to report specific data for breakdown
+        categories. For example, a <strong>Staff</strong> metric could be broken
+        down into{" "}
         <strong>
           Security Staff, Management Staff, Clinical Staff, Other Staff
```
Comment (morden35) [inline publisher/src/components/HelpCenter/Guides/SetUpMetricsGuide.tsx:62]: nit there is a space between `Other Staff .`

File: publisher/src/components/HelpCenter/Guides/SetUpMetricsGuide.tsx [hunk 4]
```diff
@@ -79,31 +74,33 @@ export const SetUpMetricsGuide = () => (
...
+            metric and its breakdown(s), if breakdowns are enabled. You will not
+            need to share data for each definition category, these are simply
+            specified so that people viewing your data understand what is or is
+            not included in your overall metric. Alternatively, you may click on{" "}
+            <strong>choose the Justice Counts definition</strong> to
+            automatically select the standard Justice Counts definition for the
+            metric/breakdown.{" "}
+          </li>
+          <li>
+            you may also add free-form text to provide even more context on the
```
Comment (morden35) [inline publisher/src/components/HelpCenter/Guides/SetUpMetricsGuide.tsx:100]: nit: do we want to capitalize 'you'?

File: publisher/src/components/HelpCenter/Guides/SuperagenciesGuide.tsx [hunk 5]
```diff
@@ -20,29 +20,56 @@ import React from "react";
...
+          <li>
+            To navigate between superagency and child agency, use the drop down
+            and search feature on the top left of the navigation bar.
+          </li>
+          <li>
+            When in the superagency view, you may enter data for both
+            superagency and child agencies. From the child agency view, you can
+            only enter data for that specific child agency. If you want to
+            upload data for multiple child agencies, you must use the bulk
+            upload option from the superagency view, not the child agency view{" "}
```
Comment (morden35) [inline publisher/src/components/HelpCenter/Guides/SuperagenciesGuide.tsx:41]: missing period

File: publisher/src/components/HelpCenter/Guides/SupervisionDisaggregationGuide.tsx [hunk 6]
```diff
@@ -23,62 +23,73 @@ export const SupervisionDisaggregationGuide = () => (
...
-            right corner and selecting Agency Settings from the dropdown.
+            Navigate to Agency Settings by clicking the icon in the top right
+            corner and selecting <strong>Agency Settings</strong> from the
+            dropdown.
           </li>
           <li>
-            Scroll down to the section titled “Supervision Populations” and
-            click “Edit populations”
+            Scroll down to <strong>Supervision Populations</strong> and click{" "}
+            <strong>Edit populations</strong>
```
Comment (morden35) [inline publisher/src/components/HelpCenter/Guides/SupervisionDisaggregationGuide.tsx:44]: nit missing period

File: publisher/src/components/HelpCenter/Guides/SupervisionDisaggregationGuide.tsx [hunk 7]
```diff
@@ -23,62 +23,73 @@ export const SupervisionDisaggregationGuide = () => (
...
-            Navigate to Metric Settings (link) and choose a metric Under the
-            “Metric” section, look for the toggle titled
-          </li>
-          <li>
-            “Disaggregated by Supervision Type” and choose either “Combined” or
-            “Disaggregated”
+            Under the <strong>Metric</strong> section, look for the toggle
+            titled <strong>Disaggregated by Supervision Type</strong> and choose
+            either <strong>Combined</strong> or
+            <strong>Disaggregated</strong>
```
Comment (morden35) [inline publisher/src/components/HelpCenter/Guides/SupervisionDisaggregationGuide.tsx:66]: missing space between or and Disaggregated

File: publisher/src/components/HelpCenter/Guides/SupervisionDisaggregationGuide.tsx [hunk 8]
```diff
@@ -23,62 +23,73 @@ export const SupervisionDisaggregationGuide = () => (
...
-        (the total funding for the agency) but disaggregated numbers for Daily
-        Population (a separate count for parole and probation).
+        Supervision agencies may oversee one or more supervision populations,
+        including: parole, probation, pretrial supervision, and other. For each
+        metric, you can choose to share the data for your supervision operations
+        as a whole, or disaggregated by your supervision populations.
+        <ol>
+          <li>
+            Specify which supervision populations your agency oversees in
+            <strong>Agency Settings</strong>. If you don’t distinguish between
```
Comment (morden35) [inline publisher/src/components/HelpCenter/Guides/SupervisionDisaggregationGuide.tsx:81]: missing space between in and Agency Settings

**Pull request merged**